.circleci/config.yml

```diff
@@ -69,6 +70,57 @@ commands:
           name: print installed packages
           command: pip freeze --all
 
+  dependencies-windows:
+    # You must still manually update the Docker image if any
+    # binary (non-Python) dependencies change.
+    description: "Check out and update Python dependencies."
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - v6-win-dependencies-{{ checksum "setup.py" }}
+
+      - run:
+          name: install python
+          command: |
+            $ErrorActionPreference = "Stop"  # exit immediately on any error
+            choco install --side-by-side -y python --version 3.8
+          shell: powershell.exe
+
+      - run:
+          name: install virtualenv
+          command: pip install virtualenv
+          shell: powershell.exe
+
+      - run:
+          name: install dependencies
+          # Only create venv if it's not been restored from cache
+          command: if (-not (Test-Path venv)) { .\ci\build_and_activate_venv.ps1 -venv venv }
+          shell: powershell.exe
+
+      - save_cache:
+          paths:
+            - .\venv
+          key: v6-win-dependencies-{{ checksum "setup.py" }}
```

Comment:
For symmetry consider changing the Linux cache to `v6-unix-dependencies-{{ checksum "setup.py" }}`